tools/pixel.md

```diff
@@ -3,51 +3,60 @@ layout: post
 permalink: /pixel
 title: Google Pixel
 icon_slug: google
+command: "Settings -> About Phone -> Regulatory labels"
```

Comment:
Other files in /tools don't use quotation marks for the command field, but I'm not sure with this one. Don't know if that's the problem.

```suggestion
command: Settings -> About Phone -> Regulatory labels
```

Reply:
@Kazzarra nope i tried it like that at first